Update arithmetic.md (#17490)
DataDog/documentation: content/en/dashboards/functions/arithmetic.md

{{< img src="dashboards/functions/arithmetic/cumsum.png" alt="cum sum function with abs" style="width:80%;">}}

## Cumulative sum in monitors

> Cumulative sum should be avoided in monitor queries, because the cumulative sum function is a visual function. When using within a dashboard or notebook, the points will reflect values based on the timeframe that is selected. This doesn’t translate well within a monitor as the monitor doesn’t have a sense of which timeframe to use.

Reviewer: ```suggestion
Cumulative sum should be avoided in monitor queries, because the cumulative sum function is a visual function. When used in a dashboard or notebook, the points will reflect values based on the selected timeframe. This doesn’t translate well in a monitor as the monitor doesn’t have a sense of which timeframe to use.
```

> In subsitute, configure [Cumulative Time Windows][1] in your monitor evaluation period.

Reviewer: ```suggestion
Instead, configure [Cumulative Time Windows][1] in your monitor evaluation period.
```

    {{< nextlink href="/dashboards/functions/smoothing" >}}Smoothing: Smooth your metric variations.{{< /nextlink >}}
    {{< nextlink href="/dashboards/functions/timeshift" >}}Timeshift: Shift your metric data point along the timeline. {{< /nextlink >}}
{{< /whatsnext >}}

> [1]: /monitors/types/configuration/?tab=thresholdalert#cumulative-time-windows

Reviewer: Just noting that this link doesn't work for me.

Reviewer: Ah, okay, in that case the link should be updated to instead point to `/monitors/configuration/?tab=thresholdalert#cumulative-time-windows`.

Author: modified!
